identity/app/controllers/SaveContentController.scala

```diff
@@ -0,0 +1,44 @@
+package controllers
+
+import com.google.inject.Inject
+import common.ExecutionContexts
+import idapiclient.IdApiClient
+import play.api.mvc._
+import actions.AuthenticatedActions
+import services.{PlaySavedArticlesService, IdRequestParser, ReturnUrlVerifier}
+import utils.SafeLogging
+
+class SaveContentController @Inject() ( api: IdApiClient,
```

Comment:
@daithiocrualaoich and I tried to convince those ID developers back in the day...